Instagram/View Controllers/CommentsViewController.h

```diff
@@ -0,0 +1,19 @@
+//
+//  CommentsViewController.h
+//  Instagram
+//
+//  Created by Megan Miller on 6/28/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "Post.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CommentsViewController : UIViewController
+
+@property (nonatomic, strong) Post *post;
```

Comment:
A couple questions that don't really apply since you're using storyboards, but are worth considering:

1) Should this be immutable? Meaning once it's set, should it ever change?

If so, rather than exposing it as a property, you could create a custom initializer where you pass it in instead. Unfortunately, since you're using nibs and Storyboards, I don't think you actually have this option. If you were doing everything manually without storyboards, you could do this. Lmk if you want to know what this looks like. Most companies don't use Interface Builder or Storyboards because they are difficult to code review and the merge conflicts are crazy.

2) If you added this to the initializer, you could consider making this `readonly` so it can be read but not changed. Or if nobody needs to read it, just remove it from the API.